DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -103,11 +106,85 @@ void FillBindingData(
 
 void DmlDispatchable::Initialize()
 {
-    THROW_IF_FAILED(m_device->DML()->CompileOperator(
-        m_operator.Get(), 
-        m_desc.executionFlags, 
-        IID_PPV_ARGS(m_operatorCompiled.ReleaseAndGetAddressOf())));
-    m_operatorCompiled->SetName(std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(m_name).data());
+    if(m_desc.compileType == DmlCompileType::DmlCompileOp)
+    {
+        m_logger->LogInfo("DmlCompileOp");
```

Comment:
I can remove if needed but thought it was useful to log which path is used

Reply:
How about removing the logging for the default case (DmlCompileOp), but printing something when it's compiled as a graph. Perhaps something more verbose like "Compiling op using IDMLDevice1::CompileGraph"